### PR TITLE
Fw compat

### PR DESCRIPTION
**Pull request by greg0ire**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Fixed
- compatibility with Symfony 4
```

### CODE REVIEW

File: src/Builder/ListBuilder.php [hunk 1]
```diff
@@ -171,7 +171,7 @@ public function buildActionFieldDescription(FieldDescriptionInterface $fieldDesc
             $fieldDescription->setTemplate('SonataAdminBundle:CRUD:list__action.html.twig');
         }
 
-        if (null === $fieldDescription->getType()) {
+        if ('_action' === $fieldDescription->getType()) {
```
Comment (core23) [inline src/Builder/ListBuilder.php]: You should check for `null` or `_action` value to stay BC.

File: src/Builder/ListBuilder.php [hunk 2]
```diff
@@ -167,7 +171,7 @@ public function buildActionFieldDescription(FieldDescriptionInterface $fieldDesc
             $fieldDescription->setTemplate('SonataAdminBundle:CRUD:list__action.html.twig');
         }
 
-        if (null === $fieldDescription->getType()) {
+        if (in_array($fieldDescription->getType(), ['null', '_action'], true)) {
```
Comment (core23) [inline src/Builder/ListBuilder.php]: `'null'`? Null is no string

Comment (greg0ire) [reply src/Builder/ListBuilder.php]: good catch!

**Comment by greg0ire** [comment]
@sonata-project/contributors please review

**Pull request merged**